packages/client/src/generation/generateClient.ts

```diff
@@ -123,17 +123,8 @@ export async function buildClient({
 
   // we only generate the edge client if `--data-proxy` is passed
   if (dataProxy === true) {
-    fileMap[path.join('edge', 'index.js')] = await JS(edgeTsClient, true)
-    fileMap[path.join('edge', 'package.json')] = JSON.stringify(
-      {
-        name: '.prisma/client/edge',
-        main: 'index.js',
-        types: '../index.d.ts',
-        browser: '../index-browser.js',
-      },
-      null,
-      2,
-    )
+    fileMap[path.join('edge.js')] = await JS(edgeTsClient, true)
```

Comment:
Can be done after the release
```suggestion
    fileMap[path.join('edge.js')] = await JS(edgeTsClient, true)
```